server/src/service.rs

```diff
@@ -235,7 +263,6 @@ impl Service {
             Arc::clone(&service.manager),
             Arc::new(Keyring::temporary(Secret::random()).await?),
         );
-        collections.push(collection.clone());
```

Comment:
why was this removed?

Reply:
This change is introduced from a separate [commit](https://github.com/bilelmoussaoui/oo7/pull/153/commits/764ddb9f36a4db498caa77cd646cebc346890525).

> Since session collection is a temporary collection without any items
we shouldn't push it to the Service Collections.

With this change we can avoid an extra iteration when looking for items.

Reply:
Yeah, thanks! Removed.